app/src/main/java/com/example/footballmanager/screens/result_screens/success_components/LiveScore.kt

```diff
@@ -0,0 +1,353 @@
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.requiredHeight
+import androidx.compose.foundation.layout.requiredSize
+import androidx.compose.foundation.layout.requiredWidth
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Badge
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonColors
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.em
+import androidx.compose.ui.unit.sp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import coil.compose.rememberAsyncImagePainter
+import com.example.footballmanager.R
+import com.example.footballmanager.screens.view_models.HomeViewModel
+
+@Composable
+fun LiveScoreTab(
+    modifier: Modifier = Modifier
+) {
+    val homeViewModel: HomeViewModel = viewModel()
+    val liveNowFixtures by remember {
+        mutableStateOf(homeViewModel.retrievingByLiveNowState)
+    }
+    val liveNowFixturesCount by remember {
+        mutableIntStateOf(liveNowFixtures.responseBody.size)
+    }
+    val lastLiveNowFixtureIndex by remember {
+        mutableIntStateOf(liveNowFixturesCount - 1)
+    }
+    val isNotEmpty by remember {
+        mutableStateOf((liveNowFixturesCount != 0))
+    }
+
+    AnimatedVisibility(visible = isNotEmpty) {
+        if (isNotEmpty) {
+            Column(
+                verticalArrangement = Arrangement.spacedBy(
+                    dimensionResource(id = R.dimen.medium),
+                    Alignment.Top
+                ),
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(
+                        vertical = dimensionResource(id = R.dimen.medium)
+                    )
+            ) {
+                LiveScoreHeader()
+                val defaultValue = stringResource(id = R.string.default_value)
+                LazyRow {
+                    for (i in 0..lastLiveNowFixtureIndex) {
+                        val item = liveNowFixtures.responseBody[i]
+                        with(item) {
+                            val leagueName = league?.name ?: defaultValue
```

Comment:
instead of creating redundant variables you can give this values directly to the arguments